app/assets/javascripts/oxalis/view/settings/user_settings_view.js

```diff
@@ -127,6 +132,11 @@ class UserSettingsView extends PureComponent<UserSettingsViewProps> {
               value={this.props.userConfiguration.displayCrosshair}
               onChange={this.onChangeUser.displayCrosshair}
             />
+            <SwitchSetting
```

Comment:
We should get rid of this code duplication at some point, the volume settings contain exactly the same settings as the plane tracing settings (minus the clipping distance). Not your job in this PR though :)

Reply:
Yeah, the custom viewport branch cleans this up, so this will be only temporary hopefully :)